README.md

```diff
@@ -2,3 +2,45 @@
 
 This is an annual meeting of interdenomination churches.
 Many anoited men of God share their view on how to get to heaven with biblical teachings.
+
+
+### screen Shot of The Conference  Projection (Introduction Section)
+
+<!-- ![Screen shot of portfolio project](./image/Portifolio-page1.JPG "Introduction Section"). -->
+
+<!-- #### Link to my portfolio (https://ol-create.github.io/Portfolio/) -->
+
+## Built With
+
+- html
+- css
+- vscode
+- git
+- github
+
```

Comment:
- [ ] Your project looks amazing, all my suggestions would be oriented towards Readme 

1. Let's kindly add a link to the deployed project
2. Let's kindly add a walk-through video according to the requirements.
3. Let's make sure The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Thank you very much @Amrhub feedback resolved :heavy_check_mark: